packages/desktop-client/src/hooks/useFeatureFlag.ts

```diff
@@ -9,6 +9,7 @@ const DEFAULT_FEATURE_FLAG_STATE: Record<FeatureFlag, boolean> = {
   customReports: false,
   spendingReport: false,
   simpleFinSync: false,
+  iterableTopologicalSort: false,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏IMO we want your new iterable solution to be enabled as the default. If we receive bug reports: we can point folks to disable this feature flag. And if there are no bug reports: a few releases later we can clean up this feature flag and just keep your new logic. WDYT?
```suggestion
  iterableTopologicalSort: true,
```

Reply:
Sure, just changed it